GM-SC-CHARTER.adoc

```diff
@@ -28,33 +33,40 @@ The TODO Group shall be composed of the General Members and a Steering Committee
+Actions of the Steering Committee may be taken at in-person meetings, via conference call, or through electronic means, including email or real-time chat. In order for any action to be effective, it shall be approved by a simple majority of the Steering Committee members participating in person and/or by conference call, when a quorum is so present. Quorum shall be met when at least five (5) members of the Steering Committee are present. Any vote may be taken without a meeting electronically but shall require a majority of the entire Steering Committee to pass.
  . *Working Groups*
-The TODO Group will host Working Groups for members to contribute to that are intended to accelerate its mission. Initial Working Groups are expected to cover topics including, but not limited to:
- * Well documented software to facilitate the management of open source software engagement by an organization
- * Blog and wiki articles providing recommendations on how to manage open source engagement by an organization
+The TODO Group will host Working Groups for everyone, including its general members to contribute to that are intended to accelerate its mission. Initial Working Groups are expected to cover topics including, but not limited to:
+ * Well documented software to facilitate the management of open source software engagement and operations by an organization
+ * Blog and wiki articles providing recommendations on how to manage open source operations and implement open source usage and contributions best practices by an organization
  . *Operating Principles*
  .. TODO member meeting mechanisms and expected frequency
- ... Member meetings may take place via electronic means or face-to-face. Face-to-face meeting locations and times will be set by the Steering Committee.
- ... Meetings will occur at least once per quarter.
- ... Details of meeting mechanisms and instructions for attendance will be published reasonably in advance to all members in good standing.
+ ... General Member meetings may take place via electronic means or face-to-face. Face-to-face meeting locations and times will be set by the Steering Committee.
+ ... General Member Meetings will occur at least once per quarter.
+ ... Details of meeting mechanisms and instructions for attendance will be published reasonably in advance to all general members in good standing.
  .. Communication mechanisms
- ... All public communication not during meetings will take place over the public group mailing list(s) accessible to members and non-members alike.
- ... All private communication not during meetings will take place over the private group mailing list for members only.
+ ... All public communication not during meetings will take place over the public group mailing list(s), real-time chats, issue trackers, or source code management systems and will be accessible to members and non-members alike.
```

Comment:
```suggestion
 ... All public communication not during meetings will take place over the public group mailing list(s), real-time chats, issue trackers, or source code management systems and will be accessible to members and non-members alike. Public communication must include all technical discussions and communications. Technical decisions of the project must not be done privately.
```

Reply:
Technical is everything related to contributing/participating in TODO project repos + meetings related to the activity of each of the repos. For instance:

* [TODO Guides are done via todogroup.org](https://github.com/todogroup/todogroup.org) and the readme file clarifies how people can contribute and start a new guide or make changes to existing ones. 
* OSPOlogy has a [contribution ladder](https://github.com/todogroup/ospology/blob/main/GOVERNANCE.md#ospology-repo-governance) to know how people can become maintainers
*  People can read the [community page](https://todogroup.org/community/) to join any of the working groups that are active, communication channels, and mailing lists
* Announcements of project releases, like the OSPO mind map are shared via public [GH OSPOlogy Discussions](https://github.com/todogroup/ospology/discussions).

Maybe we can be more specific on the kind of topics that refers to technical in the playbook, and leave a short sentence like the one suggested here?